src/parser/core/modules/CooldownDowntime.tsx

```diff
@@ -188,14 +188,14 @@ export abstract class CooldownDowntime extends Module {
 		if (gRep.cooldown === undefined) {
 			return 0
 		}
-		const maxCharges = gRep.charges || 1
+		const maxCharges = gRep.charges ?? 1
```

Comment:
numeric `0` is falsey, swapping to `??` to prevent unlikely edge cases

Reply:
A skill with 0 charges is non-sensical, from the perspective of this module, so defaulting 0 maxCharges back to 1 was intended.

Reply:
ah, kk. will fix and comment so i don't do that again